app/controllers/admin/rebate_forms_controller.rb

```diff
@@ -11,22 +11,18 @@ def generateqr
     @image_data = GenerateQrService.new(@rebate_form, current_user).generate_qr
   end
 
-  def edit
-  end
+  def edit; end
 
   # GET /admin/rebate_forms
   def index
     @name = params[:name]
 
-    @status = params[:status]
-
-    @council = current_user.council.presence
-
-    @rebate_forms = policy_scope(RebateForm).order(created_at: :asc)
+    @rebate_forms = policy_scope(RebateForm)
+                    .where(status: RebateForm::NOT_SIGNED_STATUS)
+                    .order(created_at: :asc)
 
     # filter by the search form fields
-    @rebate_forms = @rebate_forms.where("fields ->> 'full_name' iLIKE ?", "%#{params[:name]}%") if @name.present?
-    @rebate_forms = @rebate_forms.where(status: @status) if @status
+    @rebate_forms = @rebate_forms.where("fields ->> 'full_name' iLIKE ?", "%#{@name}%") if @name.present?
 
     respond_with json: @rebate_forms.to_json(include: [:property])
```

Comment:
We should just return `@rebate_forms.to_json(include: [property])` here.

Reply:
This one is different because it is still responding to an API style request